config/plugin/network_problem.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
 # This plugin checks for common network issues. Currently, it only checks
-# if the conntrack table is full. 
+# if the conntrack table is full.
```

Comment:
We should also change this comment:
"# if the conntrack table is more than 90% full."

Reply:
Thanks, update the comment.